server/src/main/java/org/opensearch/rest/action/admin/cluster/RestAddVotingConfigExclusionAction.java

```diff
@@ -111,4 +113,9 @@ AddVotingConfigExclusionsRequest resolveVotingConfigExclusionsRequest(final Rest
         );
     }
 
+    @Override
+    public List<Scope> allowedScopes() {
+        return List.of(ActionScopes.Cluster_ALL);
```

Comment:
What do you think about making this scope special so if the application has this scope it returns true instead of having an action where this might be missed on?

I think a singular ApplicationAwareSubject that implements isAllowed(scopes...) would be a good home for this logic

Reply:
Yeah that sounds good!